Cargo.toml

```diff
@@ -39,10 +39,15 @@ members = [
 # resolver = "2"
 
 [features]
-default = ["compress", "cookies"]
+default = ["compress-br", "compress-gz", "cookies"]
 
-# content-encoding support
-compress = ["actix-http/compress"]
+# content-encoding support algorithms (gz: Gzip + Deflate, br: Brotli, zstd: Zstd)
+compress-br = ["actix-http/compress-br"]
```

Comment:
I'd name it `compress-brotli`, `compress-gzip` and `compress-zstd`.

Reply:
Change done.
Actually this is what I have done first. But I feel it was too verbose :thinking: .
I agree it express better what is the underlying feature.